PL2/PL2-core/src/main/java/nl/tudelft/pl2016gr2/core/algorithms/GraphOrdererThread.java

```diff
@@ -0,0 +1,102 @@
+package nl.tudelft.pl2016gr2.core.algorithms;
+
+import nl.tudelft.pl2016gr2.model.AbstractNode;
+import nl.tudelft.pl2016gr2.model.GraphNodeOrder;
+import nl.tudelft.pl2016gr2.model.OriginalGraph;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Thread which can be used to order a graph.
+ *
+ * @author Faris
+ */
+public class GraphOrdererThread extends Thread {
+
+  private HashMap<Integer, GraphNodeOrder> orderedGraph;
+  private final OriginalGraph graph;
```

Comment:
I think we should delay this till the model classes have been refactored. I think the bubbledgraph will dissapear, so we might as well remove the graphinterface and keep just the originalgraph.